src/finch/julia.py

```diff
@@ -1,37 +1,13 @@
 import os
 
-import juliapkg
-
-
-def add_package(name: str, hash: str, version: str) -> None:
-    deps = juliapkg.deps.load_cur_deps()
-    if deps.get("packages", {}).get(name, {}).get("version", None) != version:
-        juliapkg.add(name, hash, version=version)
-        juliapkg.resolve()
-
-
-_FINCH_NAME = "Finch"
-_FINCH_VERSION = "1.2.0"
-_FINCH_HASH = "9177782c-1635-4eb9-9bfb-d9dfa25e6bce"
-_FINCH_REPO_PATH = os.environ.get("FINCH_REPO_PATH", default=None)
-_FINCH_REPO_URL = os.environ.get("FINCH_URL_PATH", default=None)
-
-if _FINCH_REPO_PATH and _FINCH_REPO_URL:
-    raise ValueError(
-        "FINCH_REPO_PATH and FINCH_URL_PATH can't be set at the same time."
-    )
-
-if _FINCH_REPO_PATH:  # Also account for empty string
-    juliapkg.add(_FINCH_NAME, _FINCH_HASH, path=_FINCH_REPO_PATH, dev=True)
-elif _FINCH_REPO_URL:
-    juliapkg.add(_FINCH_NAME, _FINCH_HASH, url=_FINCH_REPO_URL, dev=True)
-else:
-    add_package(_FINCH_NAME, _FINCH_HASH, _FINCH_VERSION)
-
+#To change the version of Finch used, see the documentation for pyjuliapkg here: https://github.com/JuliaPy/pyjuliapkg
+#Use pyjuliapkg to modify the `juliapkg.json` file in the root of this repo.
+#An example development json is found in `juliapkg_dev.json`
 import juliacall as jc  # noqa
-
-juliapkg.resolve()
```

Comment:
@willow-ahrens For the pydata/sparse `notebook` job that is failing with missing dependencies, maybe we should we have `juliapkg.resolve()` here? https://github.com/pydata/sparse/actions/runs/13648682369

Reply:
Yes. It's strange that tests pass here but fail there. It may be that resolve is only called on the first instance of loading juliacall, so if pydatasparse imports juliacall and then imports finch later, resolve doesn't get called with the new deps.